airbyte-webapp/package.json

```diff
@@ -125,26 +124,28 @@
     "@types/unist": "^2.0.5",
     "@typescript-eslint/eslint-plugin": "^5.27.1",
     "@typescript-eslint/parser": "^5.27.1",
+    "@vitejs/plugin-basic-ssl": "^1.0.1",
+    "@vitejs/plugin-react": "^3.0.1",
     "babel-jest": "^29.3.1",
     "dotenv": "^16.0.3",
-    "eslint-config-prettier": "^8.5.0",
+    "eslint-config-prettier": "^8.6.0",
     "eslint-config-react-app": "^7.0.1",
     "eslint-plugin-css-modules": "^2.11.0",
     "eslint-plugin-jest": "^26.5.3",
-    "eslint-plugin-jsx-a11y": "^6.6.1",
-    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-jsx-a11y": "^6.7.1",
+    "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-unused-imports": "^2.0.0",
     "express": "^4.18.1",
     "husky": "^8.0.1",
     "jest": "^29.3.0",
     "jest-environment-jsdom": "^29.3.1",
     "license-checker": "^25.0.1",
     "lint-staged": "^12.3.7",
-    "mini-css-extract-plugin": "^2.6.1",
+    "meow": "^9.0.0",
     "node-fetch": "^2.6.7",
+    "optionator": "^0.9.1",
```

Comment:
:information_source: Peer dependencies required by the styellint and eslint plugin for vite checker.